cmd/server/http/describe.go

```diff
@@ -168,3 +170,47 @@ func describeArtifact(ctx context.Context, payload *payload, flowSvc *flow.Servi
 		}
 	}
 }
+
+func describeLatestArtifacts(ctx context.Context, payload *payload, flowSvc *flow.Service, service string) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		if emptyString(service) {
+			requiredFieldError(w, "service")
+			return
+		}
+		values := r.URL.Query()
+		branch := values.Get("branch")
+		if emptyString(branch) {
+			branch = "master"
```

Comment:
Just an observation. This default value. When will we use the endpoint without? I'm thinking whether it should be required on the server and let the client default it? I dunno. 🤷‍♂️

Reply:
sounds good with me 👍